docs/documentation/head/connect.md

```diff
@@ -124,16 +124,18 @@ Connection conn = DriverManager.getConnection(url);
 
     It can be a PEM encoded X509v3 certificate
 
-	*Note:* defaultdir is ${user.home}/.postgresql/ in *nix systems and %appdata%/postgresql/ on windows 
+	*Note:* defaultdir is ${user.home}/.postgresql/ in *nix systems and %appdata%/postgresql/ on windows. This parameter is ignored when using PKCS-12 keyfiles.
```

Comment:
I don't believe this is correct ?

Reply:
In LibPQFactory.java, the PGProperty.SSL_CERT property is only referenced in the initPk8 method. According to what I see in the code and the previous version of the docs, the PKCS-12 format uses the p12 file for both the key and the certificate.

Reply:
ah, ok, got it. I thought *this* referred to defaultdir. Perhaps we could make that more clear ?

Reply:
Okay, I moved the defaultdir explanation to the first paragraph, and added some more detail to why the cert parameter is ignored.